ui/src/shared/utils/fromFlux.test.ts

```diff
@@ -230,4 +233,32 @@ there",5
     expect(flux.fluxGroupKeyUnion).toEqual([])
     expect((fluxLegacy.table as any).columns['_value'].data).toEqual([28])
   })
+
+  test('it crushes whatever this bug is', () => {
+    const file = path.join(__dirname, './', 'fromFlux.newline.csv')
```

Comment:
we have to use `fs` here, or else typescript starts fumbling over itself to validate the values within a string